### PR TITLE
DAOS-8032 Test: Updating the test to use auto size pool.

### PR DESCRIPTION
**Pull request by ravalsam**
Ci test is getting run on smaller size of NVMe so changing the test to use the pool % size instead of fixed.
Removed the skip as DAOS-7021 is resolved.

Quick-Functional: true
Test-tag: nvme_io iorsmall

Signed-off-by: Samir Raval <samir.raval@intel.com>

### CODE REVIEW

File: src/tests/ftest/nvme/io.yaml [hunk 1]
```diff
@@ -40,15 +40,15 @@ ior:
...
-    - [4000000000, 9000000000, 24, 25165824, 1]                  #[4G, 9G, 24B, 24M, 1]
-    - [1000000000, 9000000000, 2056, 4227136, 1]                 #[1G, 9G, 2056, 4M, 1]
-    - [6000000000, 322122547200, 134217728, 34359738368, 1]      #[6G, 300G, 128M, 32G, 1]
-    - [10000000000, 64424509440, 32, 44040192, 1]                #[10G, 60G, 32B, 42M, 1]
-    - [62277025792, 1000204886016, 2048, 1073741824, 2]          #[58G, 900G, 2k, 1G, 2]
-    - [62277025792, 1000204886016, 8, 10485760, 2]               #[58G, 900G, 8B, 10M, 2]
-    - [62277025792, 1000204886016, 33554432, 23622320128, 5]     #[58G, 900G, 32M, 22G, 5]
+#   - [scmsize%, nvmesize%, stripesize, blocksize, clientslots]
+    - ["2%", "75%", 1048576, 17179869184, 1]        #[2%, 75%, 1M, 16G, 1]
+    - ["1%", "4%", 4096, 1073741824, 1]             #[1G, 9G, 4k, 1G, 1]
```
Comment (shimizukko) [inline src/tests/ftest/nvme/io.yaml:45]:
```suggestion
    - ["1%", "4%", 4096, 1073741824, 1]             #[1%, 4%, 4k, 1G, 1]
```
Don't you want to update the rest of the comments?

**Review by shimizukko — APPROVED** [review]

File: src/tests/ftest/nvme/io.yaml (hunk 1 above)
Comment (ravalsam) [reply src/tests/ftest/nvme/io.yaml]: Good catch. Updated now.

**Comment by ravalsam** [comment]
Ci test result can be found from https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-8566/1/artifact/Functional%20Hardware%20Large/nvme/io.py/

**Review by shimizukko — APPROVED** [review]

**Review by daltonbohning — APPROVED** [review]
FYI that commit pragmas are not automatically re-used. They have to be specified for each new commit. So the last commit didn't run the test (though it was just a comment change)

**Comment by ravalsam** [comment]
> FYI that commit pragmas are not automatically re-used. They have to be specified for each new commit. So the last commit didn't run the test (though it was just a comment change)

right I did not update pragmas as it was comment change only

**Pull request merged**